app/api/v1/health.py

```
@@ -23,3 +40,135 @@ def health_check() -> HealthResponse:
+    try:
+        # Simple query to verify connection
+        db.execute(text("SELECT 1"))
+        latency = (time.time() - start) * 1000  # Convert to ms
+
+        return DependencyHealth(
+            name="postgresql",
+            status="ok",
+            latency_ms=round(latency, 2),
+        )
+
+    except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)